pandera/backends/pyspark/decorators.py

```diff
@@ -1,16 +1,21 @@
 """This module holds the decorators only valid for pyspark"""
 
 import functools
+import logging
 import warnings
+from contextlib import contextmanager
 from enum import Enum
 from typing import List, Type
 
 import pyspark.sql
 
+from pyspark.sql import DataFrame
```

Comment:
Replace the module import `import pyspark.sql` statement with similar object imports as on line 12